elasticjob-cloud/elasticjob-cloud-scheduler/src/main/java/org/apache/shardingsphere/elasticjob/cloud/scheduler/util/HttpClientUtils.java

```diff
@@ -0,0 +1,151 @@
+
+/**
+ * Http client utils.
+ */
+public class HttpClientUtils {
+    
+    /**
+     * http get request.
+     * @param url thr url
+     * @return http result
+     */
+    public static HttpResult httpGet(final String url) {
+        return httpGet(url, null, null, 3000L);
+    }
+    
+    /**
+     * http get request.
+     * @param url           thr url
+     * @param paramValues   the param values
+     * @param encoding      the encoding
+     * @param readTimeoutMs the read timeout
+     * @return http result
+     */
+    public static HttpResult httpGet(final String url, final List<String> paramValues, final String encoding, final long readTimeoutMs) {
+        HttpURLConnection conn = null;
+        try {
+            String encodedContent = encodingParams(paramValues, encoding);
+            String urlWithParam = url + (null == encodedContent ? "" : ("?" + encodedContent));
+            conn = (HttpURLConnection) new URL(urlWithParam).openConnection();
+            conn.setRequestMethod("GET");
+            conn.setConnectTimeout((int) readTimeoutMs);
+            conn.setReadTimeout((int) readTimeoutMs);
+            conn.connect();
+            String resp;
+            if (HttpURLConnection.HTTP_OK == conn.getResponseCode()) {
+                resp = IOUtils.toString(conn.getInputStream(), encoding);
+            } else {
+                resp = IOUtils.toString(conn.getErrorStream(), encoding);
+            }
+            return new HttpResult(conn.getResponseCode(), resp);
+        } catch (IOException ex) {
+            throw new HttpClientException(ex);
+        } finally {
+            if (conn != null) {
+                conn.disconnect();
+            }
+        }
+    }
+    
+    private static String encodingParams(final List<String> paramValues, final String encoding) throws UnsupportedEncodingException {
+        if (null == paramValues) {
+            return null;
+        }
+        StringBuilder sb = new StringBuilder();
+        for (Iterator<String> iter = paramValues.iterator(); iter.hasNext();) {
+            sb.append(iter.next()).append("=");
+            sb.append(URLEncoder.encode(iter.next(), encoding));
+            if (iter.hasNext()) {
+                sb.append("&");
+            }
+        }
+        return sb.toString();
+    }
+    
+    /**
+     * http post request.
+     * @param url thr url
+     * @return http result
+     */
+    public static HttpResult httpPost(final String url) {
+        return httpPost(url, null, null, 3000L);
+    }
+    
+    /**
+     * http post request.
+     * @param url           thr url
+     * @param paramValues   the param values
+     * @param encoding      the encoding
+     * @param readTimeoutMs the read timeout
+     * @return http result
+     */
+    public static HttpResult httpPost(final String url, final List<String> paramValues, final String encoding, final long readTimeoutMs) {
```

Comment:
Please do not use abbreviation, `readTimeoutMs` should be renamed as `readTimeoutMillisecond`